app/views/staff/home/_delivery_partner_organisations_table.html.haml

```diff
@@ -0,0 +1,14 @@
+%table.govuk-table#delivery_partner_organisations
+  %caption.govuk-table__caption.govuk-table__caption--m
+    Delivery parnter organisations
+  %thead.govuk-table__head
+    %tr.govuk-table__row
+      %th.govuk-table__header{scope: "col"} Name
+      %th.govuk-table__header{scope: "col"} Action
```

Comment:
Should we use translations for these? (or do we already have some we can reuse?)

Reply:
Oh no! One of my pet peeves! 😅 

Yes, your right, for consistency with our code we should (I am against translations for uses like this one generally!) - I was looking at the export tables that have been introduce lately and they didn't use the locales so I think I wanted to see what would happen if I did the same!

I'll go and see what I can do. 👍